src/wwwroot/controllers/MainCtrl.js

```diff
@@ -195,6 +195,10 @@
     $scope.isPermanentAuthed = function () {
       return auth.isAuthed() && !auth.isTemporarilyAuthed();
     };
+
+    $scope.applyActive = function (name) {
+      return $location.path().split(/[\s/]+/).pop() == name ? true : false;
```

Comment:
In general, I prefer to avoid ternary operator for boolean results. Do not change it now, but what do you think about this format for future scenarios?

```csharp
$scope.applyActive = function (name) {
    return $location.path().split(/[\s/]+/).pop() == name;
```